tests/functional/test_splinter_shadow.c

```diff
@@ -100,11 +100,12 @@ test_splinter_shadow_create(test_splinter_shadow_tree **tree,
                             platform_heap_id            hid,
```

Comment:
I wish `test_splinter_shadow_create` were also an `...init`.

But that is definitely outside the scope of this PR, so not your problem!

Reply:
Also, `hh` is now unused, so delete it.